index.html

```diff
@@ -230,52 +230,79 @@ <h4 class="work-item-title">Multi-Post Stories</h4>
       </section>
       <section class="contact">
         <p class="contact-title desktop">Let's chat!</p>
-        <p class="contact-title mobile">CONTACT US</p>
+        <p class="contact-title mobile">CONTACT ME</p>
```

Comment:
- [ ] Please change the `<p>` tag with `<hx>` like h2, h3, h4. Lets follow the best code practices